test/evergreen/compatibility_test_for_releases.sh

```diff
@@ -688,7 +688,7 @@ import_release_branches=(develop mongodb-6.0 mongodb-5.0 mongodb-4.4 mongodb-4.2
 # The 2 arrays should be adjusted over time when newer branches are created,
 # or older branches are EOL.
 newer_release_branches=(develop mongodb-6.0 mongodb-5.0 mongodb-4.4 mongodb-4.2)
-older_release_branches=(mongodb-4.2 mongodb-4.0)
+older_release_branches=(mongodb-4.2)
```

Comment:
The comment above suggests we should adjust newer and older branches over time. I'd suggest we move `mongodb-5.0` and `mongodb-4.4` to older branches, and add `mongodb-7.0` (which can be a separate ticket).

Reply:
As discussed, will be done in [WT-11210](https://jira.mongodb.org/browse/WT-11210).